pynestml/codegeneration/resources_nest/NeuronHeader.jinja2

```diff
@@ -394,16 +377,6 @@ private:
   {%- endwith -%}
   {%- endfor -%}
 
-  /* getters/setters for initial values block (excluding functions) */
-
-  {%- for init in neuron.get_non_function_initial_values_symbols() %}
-  {%- if not is_delta_kernel(neuron.get_kernel_by_name(init.name)) %}
```

Comment:
This check is not present in lines 391-395 (leftmost column of line numbers).

Reply:
Thanks for pointing this out. I had missed this. Fixed it in my latest commit.